aerogear/data-sync-server
# Postgres integration tests

Implement integration tests. What I mean by integration tests is, load memeolist schema and execute queries/mutations against it.

How to verify - method A:
* Just watch CircleCI's output integrated in Github

How to verify - method B:
* Run CircleCI locally:
```
circleci build --job integration_test
```
How to verify - method C:
* Run integration tests locally

```
docker-compose -p aerogeardatasyncserver up
npm run db:init
npm run dev
npm run test:integration
```

This PR is based on top of https://github.com/aerogear/data-sync-server/pull/29
